{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2021-38577",
  "PublicDate": "2022-03-03T22:15:00Z",
  "References": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-38577"
  ],
  "Description": "** REJECT ** DO NOT USE THIS CANDIDATE NUMBER. Reason: This CVE has been rejected as it was incorrectly assigned. All references and descriptions in this candidate have been removed to prevent accidental usage.",
  "UbuntuDescription": "",
  "Notes": [
    "mdeslaur\u003e as of 2022-11-18, upstream bug is private",
    "mdeslaur\u003e ultimately, this CVE was rejected"
  ],
  "Bugs": [
    "https://bugzilla.tianocore.org/show_bug.cgi?id=3360"
  ],
  "Priority": "medium",
  "DiscoveredBy": "",
  "AssignedTo": "",
  "Patches": {
    "edk2": {
      "bionic": {
        "Status": "not-affected",
        "Note": ""
      },
      "devel": {
        "Status": "not-affected",
        "Note": ""
      },
      "esm-apps/bionic": {
        "Status": "not-affected",
        "Note": ""
      },
      "esm-apps/xenial": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "not-affected",
        "Note": ""
      },
      "impish": {
        "Status": "ignored",
        "Note": "reached end-of-life"
      },
      "jammy": {
        "Status": "not-affected",
        "Note": ""
      },
      "kinetic": {
        "Status": "not-affected",
        "Note": ""
      },
      "trusty": {
        "Status": "ignored",
        "Note": "out of standard support"
      },
      "upstream": {
        "Status": "needs-triage",
        "Note": ""
      },
      "xenial": {
        "Status": "ignored",
        "Note": "out of standard support"
      }
    }
  },
  "UpstreamLinks": {}
}